Changelog.md

```diff
@@ -5,6 +5,20 @@ All notable Changes to the Julia package `Manopt.jl` will be documented in this
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.46] January 1, 2024
+
+### Changed
+
+* A message it printed when a line search from `LineSearches.jl` reports search failure.
```

Comment:
This is not so accurate since now we throw that error (thanks for changing that).

Reply:
Right, that was changed. Thanks for noticing!